docs/guides/integration-google-ss.md

```diff
@@ -105,20 +105,22 @@ This script is hosted on CDN, and GPT automatically loads it with the secure sig
 
 For details, see [Client-Side Integration Guide for JavaScript](integration-javascript-client-side.md).
 
-A sample application is also available for integration using the SDK for JavaScript. See [Sample Applications](#sample-applications).
+A sample implementation is also available for integration using the SDK for JavaScript. See [Sample Implementations](#sample-implementations).
 
-## Sample Applications
+## Sample Implementations
 
-The following sample applications are available to illustrate how to integrate with the Google Ad Manager secure signals feature:
+The following sample implementations are available to illustrate how to integrate with the Google Ad Manager secure signals feature:
 
-- Server-Side UID2 Integration Example:
```

Comment:
these changes are not right -  Nothing here is "Client-side". I don't think this section requires any changes

So the original copy is actually correct to me. My understanding is:

Server-Side UID2 Integration Example:
- Generate token on server side 
- Only present and present the generated token on the webpage inside the browser and store the generated token inside cookie ad-hoc and DOES NOT use UID2 Javascript SDK inside the browser. 

Client-Server UID2 SDK Integration:
- Generate token on server side 
- store the generated token inside the UID2 Javascript SDK embedded on the webpage

The integration with the google secure signals are identical for both. 

@ssundahlTTD could you confirm if my understanding is correct?

Reply:
Hi @sunnywu I've backed out this section pending input from @ssundahlTTD : merging the rest for now.